src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -150,8 +155,16 @@ protected AbstractEntityAIBasic(@NotNull final J job)
     @Override
     protected void onException(final RuntimeException e)
     {
-        Log.getLogger().info("Pausing Entity for 5 Seconds");
-        this.setDelay(EXCEPTION_TIMEOUT);
+        final int timeout = EXCEPTION_TIMEOUT * exceptionTimer;
+        // wait for longer now
+        exceptionTimer *= 2;
+        final String name = this.worker.getName();
+        final BlockPos workerPosition = worker.getPosition();
+        final String jobName = worker.getColonyJob().getName();
+        Log.getLogger().info("Pausing Entity " + name + " (" + jobName + ") at " + workerPosition + " for " + timeout + " Seconds because of error:");
+        this.setDelay(timeout);
+        // fix for printing the actual exception
+        e.printStackTrace();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1148)